design/cryptography.md

```diff
@@ -28,21 +28,22 @@ An additional complexity is that Obscuro has the requirement to support temporar
 - Generation happens when the central sequencer bootstraps the network
 - The Master Seed is shared across Validators using the process described in the [whitepaper](https://whitepaper.obscu.ro/obscuro-whitepaper/amalgamated.html#sharing-the-master-seed)
 
-## 2. Ensure Enclave Encrypted Communication
-- Client-Enclave communication is encrypted using the Obscuro Key and the Client Key
-- Clients encrypt using the Obscuro Public Key
-- Enclave responds encrypting with the Client Public Key ( also known as Viewing Keys )
-- The Obscuro Key is a Key-Pair deterministically calculated from the Master Seed + Genesis Rollup
-- The Public key will be published to the Management contract and used by all obscuro tooling ( like the wallet extension ) to encrypt transactions
-- Enclaves will determine the Private key when calculating the Master Seed + Genesis Rollup 
-  - Enclaves have the Master Seed through the attestation process
-  - Enclaves fetch Genesis Rollup through the Layer 1 blocks
+## 2. Client-Enclave Encrypted Communication
+- Client-Enclave communication is encrypted using both Obscuro Keys and the Client Keys
+- Obscuro Key and Client (Viewing Key) are both asymmetric key pairs
+  - The Private Obscuro Key is deterministically calculated from Master Seed + Genesis Rollup
+  - The Private Client Key is created by the client in any way it chooses
+  - Both the Public keys of Obscuro and Client are derived, via a one way function, from their corresponding Private keys
```

Comment:
I don't think this statement makes sense

Reply:
I wanted to make explicit the path to obtaining the Public keys. 
Without that sentence it's implicitly expected that the reader understands that Public keys are derived from Private keys.

Should I remove it, or is there a rephrasing that would make more sense ? WDYT ?